x-pack/plugins/security_solution/cypress/tasks/network/flows.ts

```diff
@@ -48,5 +48,5 @@ export const clickOnShowTopN = () => {
 
 export const clickOnCopyValue = () => {
   cy.get(COPY).first().focus();
-  cy.focused().click({ force: true });
+  cy.focused().click({ force: true }); // eslint-disable-line cypress/unsafe-to-chain-command
```

Comment:
I'm not a Cypress expert, so I'm not sure if there's a way to rewrite the code without using chaining so that we don't have to ignore this ESLint error. Feel free to suggest a fix